cnd/src/network.rs

```diff
@@ -63,6 +63,9 @@ use std::{
 };
 use tokio::{runtime::Handle, sync::Mutex};
 
+// Export comit network types while maintaining the module abstraction.
+pub use ::comit::network::{WhatAliceLearnedFromBob, WhatBobLearnedFromAlice};
```

Comment:
I wouldn't mind doing even this:

```suggestion
pub use ::comit::network::*;
```

Reply:
Do you think that it is useful to have separate abstractions for network, http_api, db etc. I get the feeling it would be because it allows us to define types in each with the same name.

Reply:
What exactly do you mean with "abstractions" here? If you are referring to modules then I would say yes and we already have them :)